src/lib/ext-activitylog.js

```diff
@@ -264,17 +276,25 @@ class Controller {
         }
       });
 
+      if (document.location.search) {
+        const updateFilter = deSerializeFilters(searchParams);
+        this.model.setFilter(updateFilter);
+        this.view.setExistedFilters(updateFilter);
+      }
+
       const existingLogs = await this.getExistingLogs();
 
       if (existingLogs.length) {
         this.handleNewLogs(existingLogs);
       }
 
-      if (filterTabId) {
-        this.view.renderHeading({ filterTabId });
+      if (this.model.filter.tabId != null) {
```

Comment:
This logic would better be placed in `setExistedFilters`.

Reply:
done :-)